main.go

```diff
@@ -57,7 +57,13 @@ func (c *component) New(ctx context.Context, conf *config) (func(context.Context
 		LogFn:           domain.LoggerFromContext,
 		StatFn:          domain.StatFromContext,
 		AssetAttributor: a,
-		Producer:        p,
+		// AttributionFailureHandler field is simply a stub, it varies user to user
+		AttributionFailureHandler: {
```

Comment:
Task: please make this a configurable component as part of the `config` struct. 

The idea is that as we continue to add different `AttributionFailureHandler` implementations to the binary, the particular one that is appropriate can be selected at runtime via environment variables.

This means creating an actual no-op implementation type the does exactly what your stub here does, making that the default implementation via the `Settings()` method, and providing an environment variable to select the implementation by name.

Reply:
oof ok. what do you mean by environment variables?